src/components/Faq/Style.js

```diff
@@ -0,0 +1,123 @@
+import styled, { keyframes } from 'styled-components';
+import { Caption, Heading4 } from '../shared';
+
+const rotateOut = keyframes`
+  from { transform: rotate(0deg);
+  }to {transform: rotate(45deg);
+  }
+`;
+
+export const FAQContainer = styled.div`
+  width: 1162px;
+  margin: auto;
+  top: 6109px;
+  left: 139px;
+  gap: 32px;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+
+  @media (max-width: 768px) {
+    padding: 8px;
+  }
+`;
+
+export const FAQHeading = styled.div`
+  width: 612px;
+  height: 64px;
+  font-size: 48px;
+  font-weight: 600;
+  line-height: 64px;
+  color: #fff;
+  margin-bottom: -35px;
+  @media (max-width: 768px) {
+    font-size: 24px;
+  }
+`;
+
+export const FAQCaption = styled(Heading4)`
+  margin-top: -30px;
+`;
+
+export const FAQQuestionContainer = styled.div`
+  width: 100%;
+  height: 80px;
+  display: inline-flex;
+  flex-direction: column;
+  justify-content: flex-start;
+  align-items: flex-start;
+  gap: 32px;
+
+  @media (max-width: 768px) {
+    gap: 16px;
+  }
+
+  @media (max-width: 768px) {
+    padding: 6px 24px;
+  }
+`;
+
+export const FAQQuestion = styled.div`
+  flex-direction: column;
+  display: flex;
+  padding: 16px 36px;
+  align-items: flex-start;
+  gap: 404px;
+  border-radius: 16px;
+  border: 2px solid var(--neutral-3, #d9d9d9);
+  gap: 36px;
+  color: #fff;
+  font-family: Prompt;
+  font-size: 18px;
+  font-weight: 500;
+  line-height: 36px;
+  width: 1162px;
+  cursor: pointer;
+
+  transition: 0.5s;
+
+  &.opened {
+    border: 2px solid var(--color-primary-blue);
+    background: rgba(0, 0, 255, 0.1);
+  }
+
+  @media (max-width: 768px) {
+    font-size: 16px;
+    border: 12px;
+  }
+`;
+
+export const FAQIcon = styled.div`
```

Comment:
For smaller screens this is moving inwards towards the question and for larger screens this is going out of the parent div. Try using a flex with justify-between.